camera_calibration_parsers/src/camera_calibration_parsers/__init__.py

```diff
@@ -1,9 +1,40 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
```

Comment:
In my testing, this works:

```suggestion
# Copyright 2016
```

Reply:
https://github.com/ros-perception/image_common/pull/317/commits/a10661f39f06abb243803502b022ee8c368ce459